templates/test/patches/control-plane-image-ci-version.yaml

```diff
@@ -11,5 +11,5 @@ spec:
         marketplace:
           publisher: cncf-upstream
           offer: capi
-          sku: k8s-1dot18dot2-ubuntu-1804
+          sku: k8s-1dot18dot8-ubuntu-1804
```

Comment:
I did a search-and-replace, but missed this comment just above the `marketplace:` line:

```
 # we use the 1.18.2 image as a workaround there is no published marketplace image for k8s CI versions.
 # 1.18.2 binaries and images will get replaced to the desired version by the script above.
```

What's that about? Does it really need to be hard-coded to 1.18.2?

Reply:
/hold

Reply:
its gets generated from the patch file in `test/patches/control-plane-image-ci-version.yaml` so that's why its in all 3 files, not sure what it means exactly, but I'm guessing the ci image is expecting a different version or there is not ci image with this version or something

Reply:
I would change the comment to 18.8

Reply:
/hold cancel

@nader-ziada I made those changes and rebased.